tests/test_api_gen.py

```diff
@@ -85,9 +85,26 @@ def test_get_attributes(token, client, schemas_simple):
                 ["b_number", "float", False, None, False],
             ],
         },
+        "unique_values": [
+            [{"text": "0", "value": 0}, {"text": "1", "value": 1}],
```

Comment:
The order here is a bit ambiguous. We are relying on several assumptions and hoping that the same logic expectation is applied. I am not a big fun of the structure we ended up with previously but we should at least be consistent.

I'd suggest we add these unique values as another element in the existing arrays. For example, another `attributeHeader` like `uniqueValues`.

```
    expected_json = {
        "attributeHeaders": ["name", "type", "nullable", "default", "autoincrement", "uniqueValues"],
        "attributes": {
            "primary": [
                ["a_id", "int", False, None, False, [{"text": "0", "value": 0}, {"text": "1", "value": 1}]],
                ["b_id", "int", False, None, False, [
                {"text": "10", "value": 10},
                {"text": "11", "value": 11},
                {"text": "21", "value": 21},
            ]],
            ],
            "secondary": [
                ["a_name", "varchar(30)", False, None, False, [
                {"text": "Raphael", "value": "Raphael"},
                {"text": "Bernie", "value": "Bernie"},
            ]],
                ["b_number", "float", False, None, False, [
                {"text": "22.12", "value": 22.12},
                {"text": "-1.21", "value": -1.21},
                {"text": "7.77", "value": 7.77},
            ]],
            ],
        },
```

Reply:
Ok, I can do that. I am currently working on refactoring the frontend to accommodate this.